fabric-particles-v1/src/main/java/net/fabricmc/fabric/api/client/particles/FabricSpriteProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.particles;
+
+import java.util.List;
+
+import net.minecraft.client.particle.SpriteProvider;
+import net.minecraft.client.texture.Sprite;
+import net.minecraft.client.texture.SpriteAtlasTexture;
+
+public interface FabricSpriteProvider extends SpriteProvider {
```

Comment:
Please add a class comment quickly explaining what it is and how it's used (referencing ParticleFactoryRegistry) and reference FabricParticleTypes for the example